src/main/java/com/theironyard/war/Game.java

```diff
@@ -0,0 +1,126 @@
+package com.theironyard.war;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * Created by Nigel on 6/24/16.
+ */
+public class Game {
+    Player player1;
+    Player player2;
+    public static final int TOPOFDECK = 0;
+    public Game(Player player1, Player player2){
+        this.player1 = player1;
+        this.player2 = player2;
+    }
+
+    /**
+     * Sets up the Deck and player's hands.
+     */
+    public void setupPlayers(){
+
+        //Setup Deck for players
+        Deck warDeck = Deck.makeStandardDeck();
+        warDeck.shuffle();
+        int deckSize = warDeck.getSize();
+
+        //Setup player hands.
+        System.out.format("There are %s Cards in the deck :)\n", deckSize);
+        while (warDeck.getSize() > 0){//for (int i = deckSize; i >= 0; i--){
```

Comment:
Watch out for commented code.  Remove it before submissions.  Production code shouldn't have the extra stuff that was forgotten in it.